packages/codemods/README.md

```diff
@@ -0,0 +1,26 @@
+# @backstage/codemods
+
+A collection of codemods for use with Backstage projects.
```

Comment:
```suggestion
A collection of codemods for use with Backstage projects. They are intended to improve and simplify large scale refactoring, upgrades of code to use new implementations, alignment to ADRs or project standards, or other refactoring.
```

I'll admit this was a new term to me, seemingly based off the FB project, but concept it clear and sounds like a great addition. Just some added reference since this is really the only place to learn about what it does after this PR is merged.